### ISSUE TITLE
Kint 3.2: "Cannot read property 'classList' of null" with RichRenderer::$folder = false;

### ISSUE THREAD

**Issue by Tilogorn** [kint-php/kint#314]
I just updated Kint from `3.0` to `3.2` with composer and suddenly got displaying issues and JS warnings in the console. I was able to create a vanilla MWE, which should be easy to reproduce:

```
$ composer require kint-php/kint
```

`index.php` in the same directory:
```
<?php

require __DIR__ . '/vendor/autoload.php';
Kint\Renderer\RichRenderer::$folder = false;

$myarray = array(
    0 => array(
        'prop1' => 'val1',
        'prop2' => 'val2',
        'prop3' => 'val3',
    ),
    1 => array(
        'prop1' => 'val12',
        'prop2' => 'val22',
        'prop3' => 'val32',
    ),
);

d($myarray);

?>
```
Visiting the page, Chrome console says:

```
10:59:17.165 index.php:2 Uncaught TypeError: Cannot read property 'classList' of null
    at Object.addClass (index.php:2)
    at initLoad (index.php:2)
addClass @ index.php:2
initLoad @ index.php:2
load (async)
runOnce @ index.php:1
(anonymous) @ index.php:2
```

Downgrading back to Kint `3.0` solves the problem. Kint `3.1` seems affected too.

**Comment by jnvsor** [comment]
Well spotted. This broke search if the folder was disabled. I'm surprised no-one noticed it before. [Fix is in the works](https://github.com/jnvsor/kint/commit/c9faaf53df10e4b8cead370d0f8676d5f2efb176)

**Comment by jnvsor** [comment]
Fixed in c9faaf53df10e4b8cead370d0f8676d5f2efb176 - released 3.2.1
